app/controllers/IndexController.scala

```diff
@@ -16,18 +16,33 @@
 
 package controllers
 
+import controllers.actions.Actions
 import javax.inject.Inject
+import models.UserAnswers
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
 import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
-import views.html.IndexView
+
+import scala.concurrent.{ExecutionContext, Future}
 
 class IndexController @Inject()(
                                  val controllerComponents: MessagesControllerComponents,
-                                 view: IndexView
-                               ) extends FrontendBaseController with I18nSupport {
+                                 actions: Actions,
+                                 repository: SessionRepository
+                               )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+
+  def onPageLoad: Action[AnyContent] = actions.authWithSession.async {
+    implicit request =>
 
-  def onPageLoad: Action[AnyContent] = Action { implicit request =>
-    Ok(view())
+      request.userAnswers match {
+        case Some(_) =>
+          Future.successful(Redirect(controllers.routes.EstateRegisteredOnlineYesNoController.onPageLoad()))
+        case None =>
```

Comment:
As the user is no longer able to have multiple registrations at a time, I _think_ that we just need to create a new set of user answers in the index controller each and everytime... I'll think over this one.